corehq/motech/serializers.py

```diff
@@ -15,15 +15,35 @@
 """
 import datetime
 import re
+from typing import Any
 
 from dateutil import parser as dateutil_parser
 
 from corehq.motech.const import (
+    COMMCARE_DATA_TYPE_BOOLEAN,
     COMMCARE_DATA_TYPE_DECIMAL,
     COMMCARE_DATA_TYPE_INTEGER,
     COMMCARE_DATA_TYPE_TEXT,
 )
 
+FALSEY_STRINGS = ('0', 'false', 'no')
+
+
+def to_boolean(value: Any) -> bool:
+    """
+    Converts truthy and falsey values, including falsey strings, to
+    boolean.
+
+    >>> to_boolean(1)
+    True
+    >>> to_boolean('No')
+    False
+
+    """
+    if isinstance(value, str) and value.lower() in FALSEY_STRINGS:
+        return False
+    return bool(value)
```

Comment:
In case it's useful:
https://github.com/dimagi/commcare-hq/blob/a4315b64b5ef5b057492f3c9244681504dca504d/corehq/ex-submodules/dimagi/utils/parsing.py#L10-L22

Reply:
Ooo! That is useful. I still want to accept any value, and not raise a ValueError, but it would be great to use an existing constant with strings people are familiar with. e6382daa7bc